rpc/src/main_rpc_client.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2023 General Motors GTO LLC
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * SPDX-FileType: SOURCE
+ * SPDX-FileCopyrightText: 2023 General Motors GTO LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <up-client-zenoh-cpp/transport/zenohUTransport.h>
+#include <up-client-zenoh-cpp/rpc/zenohRpcClient.h>
+#include <up-cpp/uuid/factory/Uuidv8Factory.h>
+#include <up-cpp/uri/serializer/LongUriSerializer.h>
+#include <chrono>
+#include <csignal>
+#include <unistd.h>
+
+using namespace uprotocol::utransport;
+using namespace uprotocol::uuid;
+using namespace uprotocol::v1;
+using namespace uprotocol::uri;
+
+bool gTerminate = false; 
+
+void signalHandler(int signal) {
+    if (signal == SIGINT) {
+        std::cout << "Ctrl+C received. Exiting..." << std::endl;
+        gTerminate = true; 
+    }
+}
+
+UPayload sendRPC(UUri &uri) {
```

Comment:
If this is sample code that you want developers to use as reference, you need to document it.